lib/ui/window.dart

```diff
@@ -171,18 +171,16 @@ enum AppLifecycleState {
   ///
   /// When the application is in this state, the engine will not call the
   /// [Window.onBeginFrame] and [Window.onDrawFrame] callbacks.
-  ///
-  /// Android apps in this state should assume that they may enter the
-  /// [suspending] state at any time.
   paused,
 
-  /// The application will be suspended momentarily.
-  ///
-  /// When the application is in this state, the engine will not call the
-  /// [Window.onBeginFrame] and [Window.onDrawFrame] callbacks.
+  /// The application is still hosted on a flutter engine but is detached from
+  /// any host views.
   ///
-  /// On iOS, this state is currently unused.
-  suspending,
+  /// When the application is in this state, the engine is running without
+  /// a view. It can either be in the progress of attaching a view when engine
+  /// was first initializes, or after the view being destroyed due to a Navigator
```

Comment:
Let's just describe exactly what the current codebase does in terms of actual behavior instead of conceptuals for now. When we change it, we can update the doc.

Reply:
Never mind, this is the widgetbindingobserver, not the widget. Please disregard.